bestbook/static/build/css/style.css

```diff
@@ -467,4 +467,38 @@ span[title] {
 
 .approval-item-buttons button {
     width: 100%;
+}
+
+/* Candidate list */
+#candidate-list {
+    list-style: none;
+}
+
+#candidate-list li {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1em;
+}
+
+.candidate-link {
+    display: flex;
+}
+
+.candidate-link img {
+    margin-right: 1em;
+}
+
+.candidate-link span {
+    align-self: center;
+}
+
+.list-delete {
+    cursor: pointer;
+    align-self: center;
+}
+
+/* Prevents selected items from autocomplete forms from
+being rendered at the bottom of the page */
+.ui-helper-hidden-accessible {
```

Comment:
@jdlrobson or @koderjoker 
When we are using the autocomplete jquery extension, the value is getting appended to the bottom of the page. Any idea how we might get around this w/o using `!important`?